README.md

```diff
@@ -4,7 +4,7 @@ The Onfido API is used to submit check requests.
 This PHP package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
 - API version: 3.0.0
```

Comment:
It should be 3.6?

Reply:
as discussed offline this is going to change in the next release. Merging as is.